src/plots/smith/smith.js

```diff
@@ -0,0 +1,1456 @@
+'use strict';
+
+var d3 = require('@plotly/d3');
+var tinycolor = require('tinycolor2');
+
+var Registry = require('../../registry');
+var Lib = require('../../lib');
+var strRotate = Lib.strRotate;
+var strTranslate = Lib.strTranslate;
+var Color = require('../../components/color');
+var Drawing = require('../../components/drawing');
+var Plots = require('../plots');
+var Axes = require('../../plots/cartesian/axes');
+var setConvertCartesian = require('../cartesian/set_convert');
+var setConvertPolar = require('./set_convert');
+var doAutoRange = require('../cartesian/autorange').doAutoRange;
+var dragBox = require('../cartesian/dragbox');
+var dragElement = require('../../components/dragelement');
+var Fx = require('../../components/fx');
+var Titles = require('../../components/titles');
+var prepSelect = require('../cartesian/select').prepSelect;
+var selectOnClick = require('../cartesian/select').selectOnClick;
+var clearSelect = require('../cartesian/select').clearSelect;
+var setCursor = require('../../lib/setcursor');
+var clearGlCanvases = require('../../lib/clear_gl_canvases');
```

Comment:
There is no `scattersmithgl` at the moment; so I guess we don't need `clearGlCanvases` as well as any other WebGL related function.

Reply:
43a3be8db7ee003ed483ec79170c499f7c29e874

Reply:
Also `redrawReglTraces` could be removed. No?